automl/automl-script/script.whizzml

```diff
@@ -39,7 +39,9 @@
 (log-info "Obtaining unsupervised models")
 ;; Returns a list of unsupervised-models from a dataset
 (define (create-unsupervised-models dataset excluded excluded-models)
-  (let (params {"excluded_fields" (field-ids-from-names excluded dataset)}
+  (let (params {"excluded_fields"
+                (concat (non-preferred dataset)
```

Comment:
what happens if non-preferred fields are added to the excuded fields by the user?

Reply:
Not sure, I will check....